sydent/http/servlets/store_invite_servlet.py

```diff
@@ -139,6 +139,12 @@ def render_POST(self, request: Request) -> JsonDict:
         for keyword in self.sydent.config.email.third_party_invite_keyword_blocklist:
             for (key, value) in args.items():
                 if keyword in value.casefold():
+                    # make sure the blocklist doesn't stomp on invite_client_location url
+                    if key == "org.matrix.web_client_location" and keyword in [
```

Comment:
We could also skip checking the location arg entirely?

Reply:
That might make more sense, yeah.

Note the current approach will mean keywords like `http://` will still trigger the block, so I'm not sure it's quite enough.
But I also wonder if we need to consider having a way to block web client locations we don't like, so it's hard to say...

Perhaps I'd be tempted to strip the protocol from the URL in `web_client_location` before applying the filter. That way if we want to block a certain domain we can still do it?

Reply:
I stripped the protocol, that seems to provide the most flexibility? Let me know what you think.